client/validators.py

```diff
@@ -49,3 +52,20 @@ def digits_of(n):
 
     if str(checksum) != value[-1]:
         raise ValidationError('Invalid ID number: Incorrect check digit')
+
+
+def parental_consent_validator(new_volunteer):
+    """
+    If volunteer age is under LEGAL_AGE, check that received parental_consent details.
+    :raise: ValidationError if volunteer is underage and given data is missing
+    parental_consent or any parental_consent details.
+    """
+    volunteer_age = (date.today() - new_volunteer['date_of_birth']).days / 365
```

Comment:
Additionaly, please validate that the user is above 16 y/o and deny the registration if they aren't

Reply:
Opened #282